gitpod-io/website
# feature: add cover images to youtube embeds

## Description

Adds cover image to Youtube Embed so we can show an image and the video starts playing the moment we click the image

Applies to /webinars/java-panel-discussion
Can be replicated to /webinars/java-remote-development in follow up PR

<img width="1001" alt="image" src="https://user-images.githubusercontent.com/5633704/194436643-4bbe87ae-68c7-465a-a405-d0ee5c6d8fcb.png">

<img width="1104" alt="image" src="https://user-images.githubusercontent.com/5633704/194436652-12e52ad4-58b4-424a-9587-3591280f009e.png">

Fixes #2853

## Related Issue(s)

Fixes #

## How to test

https://www.loom.com/share/1daaa6ed0e2e4fd1a3a90db1a2771859

Surface area of changes is any page with a YoutubeEmbed component

For pages with the new cover image 
- /webinars/java-panel-discussion
- ensure the cover image works responsively on small and large screens
- is clickable, starts playing the Youtube video immediately
- youtube video also works responsively on small and large screens

For pages without the new cover image
- /webinars/java-remote-development
- /about
- /about/team
- /screencasts/getting-started-with-gitpod
- ensure there are no regressions
- work well on mobile and large screens



## Release Notes

```release-note
```

## Documentation



<a href="https://gitpod-staging.com/#https://github.com/gitpod-io/website/pull/2867"><img src="https://gitpod-staging.com/button/open-in-gitpod.svg"/></a>
